server/ServerApp.cpp

```diff
@@ -3707,6 +3707,12 @@ void ServerApp::CheckForEmpireElimination() {
         else if (EmpireEliminated(entry.first, m_universe.Objects())) {
             entry.second->Eliminate();
             RemoveEmpireTurn(entry.first);
+            if (GetEmpireClientType(entry.second->EmpireID()) != Networking::ClientType::CLIENT_TYPE_AI_PLAYER) {
+                auto it = m_ai_client_processes.find(entry.second->PlayerName());
+                if (it != m_ai_client_processes.end()) {
+                    it->second.Kill();
+                }
```

Comment:
is it removed from the map somewhere else?

Reply:
Fixed.

Reply:
Forgot to push changes.